weave/guides/tools/weave-in-workspaces.mdx

```diff
@@ -86,3 +86,92 @@ To navigate to a workspace from the UI:
 3. If you've set up your workspace as an automatic workspace, the workspace automatically populates with data visualizations and information about your run. The trace data for your run is under the **Weave** section of the workspace. If your workspace is a manual workspace, you can add Weave panels to by clicking **Add panels** and selecting new panels from the **Weave** section of the Add Panels menu.
 
 For more information about workspaces, see [View experiments results](https://docs.wandb.ai/models/track/workspaces).
+
+## Reference W&B artifacts in a Weave trace
+
+You can reference W&B artifacts (models, datasets, checkpoints) in your Weave traces. This creates a clickable link in the Weave UI that navigates directly to the artifact's details page, helping you track which artifact versions were used during specific operations.
```

Comment:
Link to W&B Artifact landing page here?

Side note: Anything can be stored as an artifact, it's not limited to models, datasets, or checkpoints. :)